mythril_core/src/error.rs

```diff
@@ -83,6 +84,14 @@ pub enum Error {
     NotImplemented(String),
 }
 
+impl<T: TryFromPrimitive> From<TryFromPrimitiveError<T>> for Error {
+    fn from(_error: TryFromPrimitiveError<T>) -> Error {
+        Error::InvalidValue(String::from(
```

Comment:
`TryFromPrimitiveError` implements both `ToString` and `Display`. I think using one of those would provide more useful debugging info than the static string.

Reply:
ping @ntegan 

I think something like `Error::InvalidValue(format!("{}", error))` would give us more info.

Reply:
@ntegan This is the blocker for this PR